src/sentry/static/sentry/app/views/issueList/types.tsx

```diff
@@ -0,0 +1,7 @@
+import {TagValue} from 'app/types';
+
+export type TagValueLoader = (
```

Comment:
shouldn't we move it to `app/types/*`? I believe this will be used later on different files

Reply:
Right now it's only used in `issueList` files, feels more local this way